.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: python
+
+python:
+  - "2.7"
+
```

Comment:
you'll also want a 'branches' stanza so that you don't get two builds for every pull request. See example [here](https://github.com/edx/edx-analytics-data-api/blob/master/.travis.yml#L12-L14)